docs/javascript.md

```diff
@@ -220,17 +16,17 @@ Each JavaScript object should be stored in its own file with a filename reflecti
 
 __Views__ are view-specific scripts and as with the css, their file path and name should exactly mirror the view template or partial it applies to. The name of the script object should reflect the whole view path (the object in `/admin/editions/index.js` should be called `GOVUK.adminEditionsIndex`).
 
-__Modules__ are re-useable things. An example of a module would be the script for a tabbed content block. Modules should not be initialised globaly, and should only be initialised when needed, by the layout / partial which needs it (see script initialisation). If a script is only ever going to be used in one place, don't make it a module.
+__Modules__ are re-useable things. An example of a module would be the script for a tabbed content block. Modules should not be initialised globally, and should only be initialised when needed, by the layout / partial which needs it (see script initialisation). If a script is only ever going to be used in one place, don't make it a module.
 
-__Helpers__ are scripts which are loaded everywhere (such as the script which prevents forms from being submited twice).
+__Helpers__ are scripts which are loaded everywhere (such as the script which prevents forms from being submitted twice).
 
 Namespaces should be kept simple and all constructors should be under `GOVUK`. The JavaScript layer is thin for whitehall and so (at least at present) there's no need to use deeper namespaces.
 
 ## Script initialisation
 
 Scripts should be initialised with `GOVUK.init`:
 
-    GOVUK.init(GOVUK.SomeScript, {el: '.js-the-thing'});
+    GOVUK.init(GOVUK.SomeScript, { el: '.js-the-thing' })
```

Comment:
```suggestion
```js
GOVUK.init(GOVUK.SomeScript, { el: '.js-the-thing' })
\```
```

Reply:
Done - I'm assuming you didn't mean to add the backslash

Reply:
There was no way of providing the triple backticks otherwise! :)